### DISCUSSION TITLE
#关于播放的问题

### DISCUSSION THREAD

**Discussion by Akuid2333** [Richasy/Bili.Uwp#1680] [常规]
我在播放的时候，有的视频会加载不出来，比如清晰度是杜比视界的流浪地球，我的设备是Yoga pro 14s，cpu是6800hs，gpu是AMD的核显，请问是什么情况啊

**Comment by Aris-Offline** [comment]
1 GPU驱动；
2 开发者已经说过这是练习作品（dddd），代码固然有屎山的味道，也不稳定
